docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
 
 ## The Details
 
-### What is Liquid Staking?
+### A Deep Dive into the Staking Process
+Let’s take a deep dive into what happens when you deposit your SOL with Sunrise Stake
 
-Liquid staking is a method of staking that allows stakers to unstake without
-the typical 2-3 day withdrawal period.
+#### Step 1: Depositing SOL with Sunrise Stake
+The staking process begins with you depositing your SOL via Sunrise Stake’s app.
 
-Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) protocol,
-which allows liquid staking through the use of liquidity pools.
+Your SOL will be deposited in Sunrise Stake’s Treasury Account, which is controlled by a smart contract (called Treasury Controller). 
```

Comment:
This is not quite right - it is not deposited into a treasury account, it is deposited in Marinade and Blaze stake pools . The resultant pool tokens are kept in a PDA controlled by the sunrise stake smart contract. 

The treasury account and treasury controller are not involved at this point. They only get involved once the yield is extracted from the pools.